src/layer/landuse.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+import * as Color from "../constants/color.js";
+
+export const urbanizedArea = {
+  id: "urbanized-area",
+  type: "fill",
+  minzoom: 4,
+  maxzoom: 6,
```

Comment:
We aren’t using these min/max zoom level properties as much as we could. They could improve the framerate at some zoom levels.